packages/playwright/src/generators/visual-regression/generator.spec.ts

```diff
@@ -50,13 +50,23 @@ describe('playwright generator', () => {
         addStacksAttributes(appTree, '');
     });
 
+    it('should error if the project does not supported', async () => {
+        const options: VisualRegressionGeneratorSchema = {
+            project: 'non-supported-project',
```

Comment:
We have not mocked this as a return value for getProjects(), so would error before getting to the code we want to check
```suggestion
            project: 'test',
```